terraform/projects/app-ecs-albs/main.tf

```diff
@@ -0,0 +1,145 @@
+/**
+* ## Project: app-ecs-albs
+*
+* Create ALBs for the ECS cluster
+*
+* This may be merged back in to the nodes project
+*
+*/
+
+variable "additional_tags" {
+  type        = "map"
+  description = "Stack specific tags to apply"
+  default     = {}
+}
+
+variable "aws_region" {
+  type        = "string"
+  description = "AWS region"
+  default     = "eu-west-1"
+}
+
+variable "remote_state_bucket" {
+  type        = "string"
+  description = "S3 bucket we store our terraform state in"
+  default     = "ecs-monitoring"
+}
+
+variable "stack_name" {
+  type        = "string"
+  description = "Unique name for this collection of resources"
+  default     = "ecs-monitoring"
+}
+
+# locals
+# --------------------------------------------------------------
+
+locals {
+  default_tags = {
+    Terraform = "true"
+    Project   = "app-ecs-albs"
+  }
+}
+
+# Resources
+# --------------------------------------------------------------
+
+## Providers
+
+terraform {
+  required_version = "= 0.11.7"
+
+  backend "s3" {
+    key = "app-ecs-albs.tfstate"
+  }
+}
+
+provider "aws" {
+  version = "~> 1.14.1"
+  region  = "${var.aws_region}"
+}
+
+## Data sources
+
+data "terraform_remote_state" "infra_networking" {
+  backend = "s3"
+
+  config {
+    bucket = "${var.remote_state_bucket}"
+    key    = "infra-networking.tfstate"
+    region = "${var.aws_region}"
+  }
+}
+
+data "terraform_remote_state" "infra_security_groups" {
+  backend = "s3"
+
+  config {
+    bucket = "${var.remote_state_bucket}"
+    key    = "infra-security-groups.tfstate"
+    region = "${var.aws_region}"
+  }
+}
+
+## Resources
+
+resource "aws_lb" "monitoring_external_alb" {
+  name               = "${var.stack_name}-ext-alb"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = ["${data.terraform_remote_state.infra_security_groups.monitoring_external_sg_id}"]
+
+  subnets = [
+    "${element(data.terraform_remote_state.infra_networking.public_subnets, 1)}",
+    "${element(data.terraform_remote_state.infra_networking.public_subnets, 2)}",
```

Comment:
Is this the right number of subnets?

Reply:
To start with you probably only need one but it should be in multiple AZs, I'd say 3, by the end of the beta.

Reply:
Thanks, will go 1 AZ for the moment then.